handlers/update_resource.go

```diff
@@ -0,0 +1,57 @@
+package handlers
+
+import (
+	"log"
+
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/sul-dlss-labs/taco"
+	"github.com/sul-dlss-labs/taco/generated/models"
+	"github.com/sul-dlss-labs/taco/generated/restapi/operations"
+	"github.com/sul-dlss-labs/taco/persistence"
+)
+
+// NewDepositResource -- Accepts requests to create resource and pushes them to Kinesis.
+func NewUpdateResource(rt *taco.Runtime) operations.UpdateResourceHandler {
+	return &updateResourceEntry{rt: rt, repository: rt.Repository()}
+}
+
+type updateResourceEntry struct {
+	rt         *taco.Runtime
+	repository persistence.Repository
+}
+
+// Handle the update resource request
+func (d *updateResourceEntry) Handle(params operations.UpdateResourceParams) middleware.Responder {
+	resource, err := d.repository.GetByID(params.ID)
+
+	if err == nil {
+		if err := d.updateResource(resource.ID, params); err != nil {
+			// TODO: handle this with an error response
```

Comment:
Now that we have #184 I think we can remove this as a TODO.

Reply:
k